types/boolslice_test.go

```diff
@@ -0,0 +1,50 @@
+package types
+
+import (
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestBoolSlice(t *testing.T) {
+	Convey("Given BoolSlice", t, func() {
+		Convey("valid value", func() {
+			v := []bool{true, false, true}
+			nullBoolSlice := BoolSlice(v)
```

Comment:
Why using nullBoolSlice in every tests, even when there is no involved null/nil values ?
(same for all tests of all new types).

Reply:
At first I also added `Null*` types to these new types, but that was redundant as slices can be `nil`. Those `null...` named variables are spoils from that try. All "`null`" can be renamed to "`v`" so it will become for example `vBoolSlice` for `BoolSlice`. Thanks.